cypress/e2e/links.spec.js

```diff
@@ -45,8 +45,8 @@ describe('test link marks', function() {
 				.type('google.com')
 
 			cy.getContent()
-				.find('a[href*="google.com"]')
-				.click({ force: true })
+				.get('a[href*="google.com"]')
+				.expect([])
 
 			cy.get('@winOpen')
```

Comment:
This part is not needed anymore as there is no link, thus no windows is opened.

Reply:
Removed!